.github/workflows/tests.yml

```diff
@@ -44,7 +44,9 @@ jobs:
         ln -s "$HOMEBREW_REPOSITORY" "$GITHUB_WORKSPACE"
         git fetch --tags origin "${{github.sha}}"
         git checkout --force -B master FETCH_HEAD
-        brew update-reset Library/Taps/homebrew/homebrew-core
+        for tap in Library/Taps/homebrew/*; do
```

Comment:
Can this be limited to just `bundle`, `services` and `test-bot`?

Reply:
Sure